client/web/compose/src/views/Public/Pages/Records/View.vue

```diff
@@ -119,7 +139,15 @@ export default {
 
     title () {
       const { name, handle } = this.module
-      return this.$t('page:public.record.view.title', { name: name || handle, interpolation: { escapeValue: false } })
+      let titlePreffix
```

Comment:
You can set default to 'view' and then you don't need the else.

Plus typo on prefix! (I suggest you install the spellchecker extension)